scrapd/core/apd.py

```diff
@@ -217,7 +217,27 @@ def parse_details_page_notes(details_page_notes):
             first_cap = index
             break
 
-    return squished[first_cap:]
+    # Remove HTML tags.
+    no_html = re.sub(re.compile('<.*?>'), '', squished[first_cap:])
+
+    # Remove tabs and, if subjects included, remove.
+    remove_subjects = re.split(r'\s{2,}', no_html)
+
+    # Demographic info is usually only included in subject description.
+    # DOB would be better, but that is sometimes missing.
+    for segment in remove_subjects:
+        if 'male' in segment or 'female' in segment:
```

Comment:
The `in` keyword does partial match, so looking for `male` is enough to cover both cases.